### DISCUSSION TITLE
part-ii/2019-128/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#239] [Question comments]
Part II 2019 Representation Theory: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2019-128/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(c) The Pauli matrices form a basis. For (iii) surjectivity, I calculated the action of $\begin{pmatrix}u&v\\ -\overline{v}&\overline{u}\end{pmatrix}$ on $\begin{pmatrix}i&0\\0&i\end{pmatrix}$, for $u = \cos\theta \left(\cos\phi + i\sin\phi\right)$, $v = \sin\theta \left(\cos\phi + i\sin\phi\right)$.

</details>
